lib/fs-loader.js

```
@@ -140,7 +140,9 @@ const load = async (dir, options, config) => {
   const schemas = options.preprocessor
     ? options.preprocessor(flatten(results))
     : flatten(results);
-  return Metaschema.create(schemas, config);
+  const [err, ms] = Metaschema.create(schemas, config);
```

I'd prefer to throw an error from `Metaschema.create`.